src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,65 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+  max-width: $calendarSize;
+  padding: $calendarPadding;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
The calednar should be centered both horizontally and vertically. You could get rid of these margins nad paddings and simply add flex to body (remember about justify/align and height) ;)